pandas/core/arrays/categorical.py

```diff
@@ -2237,21 +2237,20 @@ def unique(self):
         order of appearance.
 
         >>> pd.Categorical(list('baabc'))
-        [b, a, c]
-        Categories (3, object): [b, a, c]
+        [b, a, a, b, c]
+        Categories (3, object): [a, b, c]
 
         >>> pd.Categorical(list('baabc'), categories=list('abc'))
-        [b, a, c]
-        Categories (3, object): [b, a, c]
+        [b, a, a, b, c]
+        Categories (3, object): [a, b, c]
 
         An ordered Categorical preserves the category ordering.
 
-        >>> pd.Categorical(list('baabc'),
-        ...                categories=list('abc'),
-        ...                ordered=True)
-        [b, a, c]
+        >>> pd.Categorical(list('baabc'), categories=list('abc'), ordered=True)
+        [b, a, a, b, c]
         Categories (3, object): [a < b < c]
 
+
```

Comment:
do we need two lines here or just one?

Reply:
I think one.

---

Besides "See also" header, should come before the "Examples" header.

Ill fix that